src/renderer/components/conversations/Emojify.js

```diff
@@ -1,33 +1,19 @@
 const React = require('react')
 const classNames = require('classnames')
-const is = require('@sindresorhus/is')
```

Comment:
We still use this elsewhere? Or can it be removed now?

Reply:
![screenshot from 2019-01-05 20-46-28](https://user-images.githubusercontent.com/308049/50728497-0106ce80-112b-11e9-8ece-9d78b0559667.png)

Doesn't seem like it.

Reply:
I'll remove it